EPRN.Portal/Helpers/HomeServiceFactory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EPRN.Common.Enums;
+using EPRN.Portal.Helpers.Interfaces;
+using EPRN.Portal.Services.Interfaces;
+
+namespace EPRN.Portal.Helpers
+{
+    public class HomeServiceFactory : IHomeServiceFactory
+    {
+        private readonly IHomeService exporterHomeService;
+        private readonly IHomeService reprocessorHomeService;
+        private readonly IHomeService exporterAndReprocessorHomeService;
+        private readonly IWasteService wasteService;
+        private readonly IUserRoleService userRoleService;
+
+        public HomeServiceFactory(IHomeService exporterHomeService, IHomeService reprocessorHomeService, IHomeService exporterAndReprocessorHomeService, 
+            IWasteService wasteService, IUserRoleService userRoleService)
```

Comment:
Also, has this actually been tested? I don't think the DI can differentiate between the 3 classes inferred by IHomeService. I believe they will just end up being the most recent registration of IHomeService.